keyboards/iris/keymaps/fate/readme.md

```diff
@@ -0,0 +1,19 @@
+# Fate Iris Layout
+
+This keymap is configured for Pro Micro(s) with QMK-DFU bootloader flashed. Please refer to [this guide](https://www.reddit.com/r/olkb/comments/8sxgzb/replace_pro_micro_bootloader_with_qmk_dfu/) for further details on ISP flashing your Pro Micro.
+
+To generate this bootloader, run:
+
+    make iris/rev2:fate:bootloader
```

Comment:
For this, `:production` is better.  It generates both the firmware and the bootloader, actually, AND it creates a HEX that has both the bootloader *and* the firmware.  So you ISP flash the production HEX, and the board is ready to use immediately

Reply:
I was using male jumper wires directly onto the Iris PCB through-holes for ISP programming without any soldering. So, I had to apply pressure between the jumper wires and PCB to keep the connection secure. Thus, I generated the bootloader only, because I wanted to minimize the time spent of ISP flashing. I'll update this anyway so thanks!